src/dbnode/namespace/convert_test.go

```diff
@@ -387,6 +393,7 @@ func assertEqualMetadata(t *testing.T, name string, expected nsproto.NamespaceOp
 	require.True(t, expectedSchemaReg.Equal(observed.Options().SchemaHistory()))
 
 	assertEqualRetentions(t, *expected.RetentionOptions, opts.RetentionOptions())
+	assertEqualStagingState(t, expected.StagingState, opts.StagingState())
```

Comment:
Is this correct? One of these is `nsproto.StagingStatus_READY` and the other is `nsproto.StagingStatus_INITIALIZING`

Reply:
Ahh, think you wanna expand the diff a bit. This is just a generic helper, `assertEqualMetadata`, for comparing a proto object to a golang one.